lib/exchange/provider/letsexchange_exchange_provider.dart

```diff
@@ -0,0 +1,259 @@
+import 'dart:convert';
+import 'dart:developer';
+
+import 'package:cake_wallet/.secrets.g.dart' as secrets;
+import 'package:cake_wallet/exchange/provider/exchange_provider.dart';
+import 'package:cake_wallet/exchange/exchange_provider_description.dart';
+import 'package:cake_wallet/exchange/limits.dart';
+import 'package:cake_wallet/exchange/trade.dart';
+import 'package:cake_wallet/exchange/trade_not_created_exception.dart';
+import 'package:cake_wallet/exchange/trade_request.dart';
+import 'package:cake_wallet/exchange/trade_state.dart';
+import 'package:cake_wallet/exchange/utils/currency_pairs_utils.dart';
+import 'package:cw_core/crypto_currency.dart';
+import 'package:http/http.dart' as http;
+
+class LetsExchangeExchangeProvider extends ExchangeProvider {
+  LetsExchangeExchangeProvider() : super(pairList: supportedPairs(_notSupported));
+
+  static const List<CryptoCurrency> _notSupported = [];
+
+  static const apiKey = secrets.letsExchangeBearerToken;
+  static const _baseUrl = 'api.letsexchange.io';
+  static const _infoPath = '/api/v1/info';
+  static const _infoRevertPath = '/api/v1/info-revert';
+  static const _createTransactionPath = '/api/v1/transaction';
+  static const _createTransactionRevertPath = '/api/v1/transaction-revert';
+  static const _getTransactionPath = '/api/v1/transaction';
+
+  static const _affiliateId = '';
```

Comment:
please also add the affiliateId as a secret

Reply:
done